application_form/models/application.py

```diff
@@ -85,7 +73,7 @@ class ApplicationApartment(models.Model):
         Application, on_delete=models.CASCADE, related_name="application_apartments"
     )
     apartment_uuid = models.UUIDField(verbose_name=_("apartment uuid"))
-    priority_number = IntegerPGPPublicKeyField(_("priority number"))
+    priority_number = models.IntegerField(_("priority number"))
 
     class Meta:
         unique_together = [("application", "priority_number")]
```

Comment:
I wonder if this actually ever worked before. Discovered some failing tests related to this

```
=========================== short test summary info ============================
ERROR application_form/tests/test_export_service.py::test_export_applicants - django.db.utils.IntegrityError: duplicate key value violates unique constraint "application_form_applica_application_id_priority__89530de7_uniq"
DETAIL:  Key (application_id, priority_number)=(116, 1) already exists.
ERROR application_form/tests/test_export_service.py::test_export_applicants_and_secondary_applicants - django.db.utils.IntegrityError: duplicate key value violates unique constraint "application_form_applica_application_id_priority__89530de7_uniq"
DETAIL:  Key (application_id, priority_number)=(117, 5) already exists.
ERROR application_form/tests/test_export_service.py::test_csv_output - django.db.utils.IntegrityError: duplicate key value violates unique constraint "application_form_applica_application_id_priority__89530de7_uniq"
DETAIL:  Key (application_id, priority_number)=(160, 2) already exists.
ERROR application_form/tests/test_export_service.py::test_csv_non_ascii_characters - django.db.utils.IntegrityError: duplicate key value violates unique constraint "application_form_applica_application_id_priority__89530de7_uniq"
DETAIL:  Key (application_id, priority_number)=(161, 3) already exists.
ERROR application_form/tests/test_export_service.py::test_write_csv_file - django.db.utils.IntegrityError: duplicate key value violates unique constraint "application_form_applica_application_id_priority__89530de7_uniq"
DETAIL:  Key (application_id, priority_number)=(162, 3) already exists.
FAILED apartment/tests/api/test_views.py::test_project_detail_apartment_reservations_multiple_winning - django.db.utils.IntegrityError: duplicate key value violates unique constraint "application_form_applica_application_id_priority__89530de7_uniq"
DETAIL:  Key (application_id, priority_number)=(35, 1) already exists.
FAILED application_form/tests/api/test_project_reservation_view.py::test_list_project_reservations_get - django.db.utils.IntegrityError: duplicate key value violates unique constraint "application_form_applica_application_id_priority__89530de7_uniq"
DETAIL:  Key (application_id, priority_number)=(308, 4) already exists.
FAILED application_form/tests/api/test_project_reservation_view.py::test_list_project_reservations_get_without_lottery_data - django.db.utils.IntegrityError: duplicate key value violates unique constraint "application_form_applica_application_id_priority__89530de7_uniq"
DETAIL:  Key (application_id, priority_number)=(309, 3) already exists.
FAILED invoicing/tests/test_migrations.py::test_foo - django.db.utils.IntegrityError: null value in column "type" violates not-null constraint
DETAIL:  Failing row contains (49, 2024-08-18 19:10:02.196468+00, null, null, , null, TEMP-3b3e8452-6d72-4ea5-9178-5555c04892b1, null, 730000001, null, null, 2024-08-18 19:10:02.196472+00, ).
====== 4 failed, 419 passed, 4449 warnings, 5 errors in 150.73s (0:02:30) ======
```